packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -91,16 +91,37 @@ export class HeapDumpService {
               innerExtent.symbols && innerExtent.symbols.length > 0
                 ? innerExtent.symbols[0]
                 : undefined;
+            const className = symbolName
+              ? this.logContext.getUtil().substringUpToLastPeriod(symbolName)
+              : undefined;
             if (symbolName && frameInfo.locals.has(symbolName)) {
               const localVar = frameInfo.locals.get(
                 symbolName
               ) as ApexVariableContainer;
               this.updateVariableContainer(localVar, innerExtent.value);
+            } else if (
+              symbolName &&
+              className &&
+              this.logContext.getStaticVariablesClassMap().has(className)
```

Comment:
Takes advantage of the static field name in a heapdump varaible introduced with server side changes.